src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Services/CommitmentsService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SFA.DAS.Commitments.Api.Client;
+using SFA.DAS.EAS.Domain.Interfaces;
+
+namespace SFA.DAS.EAS.Infrastructure.Services
+{
+    public class CommitmentsService : ICommitmentsService
+    {
+        private readonly ICommitmentsApi _commitmentsApi;
+
+        public CommitmentsService(ICommitmentsApi commitmentsApi)
```

Comment:
We should probably have a quick chat about this. I can see why you've introduced it. We should update the other commands to use the service too if that's how we want to go.